drivers/pinmux/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-zephyr_library()
-
```

Comment:
Why is this deleted

Reply:
because used to be zephyr_sources_ifdef, but now zephyr_library_sources_ifdef no need to have zephyr_library